tomviz/PresetDialog.cxx

```diff
@@ -0,0 +1,39 @@
+/* This source file is part of the Tomviz project, https://tomviz.org/. 
+   It is released under the 3-Clause BSD License, see "LICENSE". */
+
+#include "PresetDialog.h"
+#include "PresetModel.h"
+#include "ui_PresetDialog.h"
+
+#include <QTableView>
+#include <QtWidgets>
```

Comment:
We avoid include entire modules, like QtWidgets. This can really slow down compilation. Prefer to forward declare what we can in the header, and then include the headers for the classes used.